canonical-web-and-design/build.snapcraft.io
# Update to Vanilla 2.0

## Done

- Updated Vanilla to latest version
- Fixed issues with header navigation and footer
- Footer updated to latest version from snapcraft.io
- Some styling updates in Vanilla dependent components

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Make sure everything renders correctly
- Check if header navigation works ok with dropdown
- Check footer style
- Check if there is no visible width difference (jumping) when switching between snapcraft.io and build
- Click around different pages (add repo, repository builds, build details) and check if everything looks ok


## Issue / Card

Fixes #1232 

## Screenshots

<img width="1169" alt="Screenshot 2019-08-13 at 15 55 53" src="https://user-images.githubusercontent.com/83575/62947478-e09aa300-bde2-11e9-8cc3-85bb0b235e82.png">
